main/tests.py

```diff
@@ -59,3 +59,10 @@ def test_invalid_login_info(self, client, make_user):
         response = client.post('/login/', credentials, follow=True)
         assert response.status_code == 200
         assert b"Please enter a correct email" in response.content
+
+    def test_logout_link_functionality(self, client, make_user):
+        sample_user = make_user("test1@gmail.com", "testy", "test", "1995-05-05", "test123")
+        client.login(email="test1@gmail.com", password="test123")
+        response = client.get('/logout/')
+        logged_user = response.wsgi_request.user
```

Comment:
Just for my knowledge...
`response.wsgi_request.user` spouse to be `None`?

Reply:
Hey @tamirho 
Not `None`, But a special 'Anonnymous user' object when no user is logged in.
You can read more about it here:
https://docs.djangoproject.com/en/3.2/ref/contrib/auth/#anonymoususer-object

Reply:
Nice! Thanks